lib/components/MenuItem/MenuItem.docs.tsx

```diff
@@ -20,6 +20,12 @@ const docs: ComponentDocs = {
   migrationGuide: true,
   description: (
     <Stack space="large">
+      <Text>
+        Follows the{' '}
+        <TextLink href="https://www.w3.org/TR/wai-aria-practices-1.1/#menu">
```

Comment:
Would this be better to reference the [Menu Button spec](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton)?